### ISSUE TITLE
Consider enforcing Plugin/WordPress Version

### ISSUE THREAD

**Issue by kraftner** [akirk/friends#29]
Trusting a friend as a person is one thing, but trusting his care for maintaining a WordPress installation is obviously another thing. As soon as your content leaves your site and starts floating around your network leakage is bound to the weakest link in that network. This is inherit to the concept of the Plugin and generally is a fine compromise.

Still I wonder if putting some more or less strong pressure on peers to keep WordPress and the Friends Plugin updated might be a good idea.

Currently the plugin [already announces its version](https://github.com/akirk/friends/blob/eb046e10df864f8f686eba8d2fdfd237698ca5a1/class-friends-rest.php#L153). So what about checking the remote version and refusing befriending or content-distribution or at least adding some sort of warning or notice?

I haven't really thought this through in terms of (technical) feasibility and usability so for now I am just leaving it as a note that popped up in my head while watching your talk at yesterdays WPVienna Meetup.

**Comment by jeffpaul** [comment]
@kraftner what's the concern if a friends' site has a WordPress or Friends version that's suboptimally out of date?  How would that impact your site?  Besides wanting to recommend to friends/connections that they keep things up-to-date, what's the primary concern here?

**Comment by kraftner** [comment]
If I publish semi-private content that is only meant to be distributed among a small circle of friends/family/etc. handing it to a site that is badly-maintained and potentially easy to hack might increase the risk of that content becoming public. At least that seem to have been my thoughts here ... 6 years ago. 

Thinking about it again today I probably wouldn't create this issue again since that is just an inevitable, inherit fact of distributed systems.

**Comment by jeffpaul** [comment]
Yeah, I think sending semi-private content outside your site is almost always a risky endeavour though I can see wanting to try and at least design around that somehow within the Friends plugin.  So perhaps a dismissable admin notice for anyone on (a somehow defined) outdated version of WordPress or Friends along the lines of:

> Hey, please consider updating your WordPress and Friends version so that <friends listed here> can reliably continue sharing content with you.  Don't let your friends down, update today!

...with respective links to the WordPress Updates page linked in there perhaps on the "update today" reference?

**Comment by kraftner** [comment]
Yeah, maybe. But not sure if it is worth the complexity here.
